README.md

```diff
@@ -1,9 +1,111 @@
-# Replication
+# Replication [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=replication&metric=alert_status)](https://sonarcloud.io/dashboard?id=replication) [![Known Vulnerabilities](https://snyk.io/test/github/connexta/replication/badge.svg)](https://snyk.io/test/github/connexta/replication)
 
+## Overview
+Replication is the process of creating a copy of a subset of data and storing it on another DDF or HDFS based system. Data can be pulled from a remote DDF and saved to another DDF or HDFS system. Metacards produced by replication are marked with a "Replication Origins" attribute and a tag of "replicated". Replication will automatically start transferring data once an admin creates a replication configuration.
+
+## Installing Replication
+Replication is not installed by default with a standard installation. There are several installation options available.
+
+#### Installing Via Admin Console
+An administrator can install the replication feature by following these steps:
+* Navigate to the Admin Console.
+* Select the System tab.
+* Select the Features tab.
+* Locate the replication feature.
+* Select the Install arrow for the replication feature.
+* The status will change from Uninstalled to Installed.
+
+#### Installing Via Kar Feature File
+An administrator can install the replication feature by following the following steps:
+* Locate and download the desired replication artifact kar file.
+* On a running and configured DDF instance, place the kar file in the ${DDF_HOME}/deploy directory.
+* Confirm the feature is running in the Admin Console or the Karaf command line interface.
+
+## Configuring Replication Nodes
+This version of Replication supports nodes created for DDF based applications as well as webHDFS based systems.
```

Comment:
`...DDF-based applications as well as webHDFS-based systems.`

Also, for consistency should it just say HDFS instead of webHDFS here?  HDFS is used at the top of this readme.